gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Mon Dec 05 11:54:51 GMT 2016
+#Fri Jun 23 08:01:21 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-bin.zip
-org.gradle.daemon=false
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.1-all.zip
```

Comment:
This change has been done automatically, I suspect that Gradle itself is fixing some ¿deprecated? paths.